src/pages/workspaces/workspace/notebooks/modals/CloudEnvironmentModal.js

```diff
@@ -202,7 +217,7 @@ export const CloudEnvironmentModal = ({
           shape: 'pause',
           toolLabel,
           onClick: () => stopApp(toolLabel),
-          disabled: busy || !canCompute,
+          disabled: busy || !canCompute || toolLabel === tools.Azure.label, //TODO: Azure stop/start,
```

Comment:
You can avoid this by making the tool not support Pause (like Cromwell). `isPauseUnsupported: true`

My goal was to keep tool-specific logic within the tools constant to avoid sprinkling conditionals throughout the rendering code.